syntax="proto3";

package allocator.v1;

service Allocator {
    rpc RequestSharedMemory(MemoryRequest) returns (MemoryReply);
}

message MemoryRequest {
    
}

message MemoryReply {
    string shm_name = 1;
    int32 size = 2;
}